gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/SubscriberApi.java

```diff
@@ -573,7 +550,7 @@ public void modifyAckDeadline(ModifyAckDeadlineRequest request) {
    * @param ackIds The acknowledgment ID for the messages being acknowledged that was returned
    * by the Pub/Sub system in the `Pull` response. Must not be empty.
    */
-  public void acknowledge(String subscription, List<String> ackIds) {
+  public final void acknowledge(String subscription, List<String> ackIds) {
```

Comment:
I see, but I am not sure if we can make every such parameter Iterable type. The List type is auto- determined by looking at "repeated" field in the proto files during code generation. So it is hard to make some cases "Iterable" and the others "List". And we do not know if the order matters or not.